Lua/plugin.lua

```diff
@@ -1,5 +1,5 @@
 function OnSetText(uri, text)
-	if string.find(uri, '.vscode') or text:sub(1, 8) == "---@meta" then return end
+	if string.find(uri, '[\\/]%.vscode[\\/]') or text:find("%-%-%-@meta") then return end
```

Comment:
The check for `---@meta` at the top of the file is to avoid having to parse large text files, and it should always be the first thing written. If there's some files where it's several lines down (i.e. comments) then I'd rather just move it to the top.